crates/aptos/src/account/fund.rs

```diff
@@ -36,6 +36,20 @@ pub struct FundWithFaucet {
     pub(crate) profile_options: ProfileOptions,
 }
 
+impl FundWithFaucet {
+    /// Create a new instance of this command with the provided amount.
+    /// This is useful for testing. If `account` is not provided, the default account will be funded.
+    pub fn default_with_amount(amount: u64, account: Option<AccountAddress>) -> Self {
```

Comment:
Use `Default` trait here?

Reply:
also do we want to make the amount to be optional as well?

Reply:
updated. use partial `Default` + util under indexer.